apps/meteor/client/lib/convertTimeUnit.ts

```diff
@@ -0,0 +1,58 @@
+export enum TIMEUNIT {
+	days = 'days',
+	hours = 'hours',
+	minutes = 'minutes',
+}
+
+const isValidTimespan = (timespan: number): boolean => {
+	if (Number.isNaN(timespan)) {
+		return false;
+	}
+
+	if (!Number.isFinite(timespan)) {
+		return false;
+	}
+
+	if (timespan < 0) {
```

Comment:
Would a timespan of 0 be a valid value? 🤔

Reply:
It already is today. It'll delete every message once the cron runs.